libs/blocks/table/table.js

```diff
@@ -97,6 +97,34 @@ function handleHeading(table, headingCols) {
   });
 }
 
+function handleAddOnContent(table) {
+  const addOnKey = 'ADDON';
+  const addOns = [...table.querySelectorAll('.section-row-title')]
+    .filter((row) => row.innerText.includes(addOnKey));
+  if (!addOns.length) return;
+  table.classList.add('has-addon');
+  addOns.forEach((addOn) => {
+    const addOnRow = addOn.parentElement;
+    addOnRow.remove();
+    const [position, order, style] = addOn.innerText.split('_')
```

Comment:
Any reason you are using `_` for authoring of the key? I don't think we should introduce a new pattern from what I can see. We've only used `-` or spaces between words. Hero-marquee uses `-` for it's row keys. 

Doing a quick test and replacing `_` with `-` and the addons are not added.
![image](https://github.com/user-attachments/assets/d611bbd8-0327-4b44-ad1b-5c82e21fd271)

Reply:
Should also sanitize the key to lowercase. Currently if it's not authored in all caps it breaks too.

[Example page](https://sartxi-mwpw-157999-table-addon--milo--adobecom.hlx.page/drafts/rclayton/test/table/add-on-greentext)

![image](https://github.com/user-attachments/assets/322261da-3eea-43c0-af23-e5b0ce8a5fc0)

Reply:
@rgclayton okay I agree, I have updated the authoring and sanitized the keys so you can use lowercase